app/controllers/idv/welcome_controller.rb

```diff
@@ -23,6 +23,11 @@ def show
     end
 
     def update
+      if !FeatureManagement.idv_available?
+        redirect_to idv_unavailable_url
+        return
+      end
```

Comment:
@matthinz @jmhooper I had to add this manual check here, because the feature spec for "prevents a user who reset their password from reactivating profile with no personal key" seemed to assume the user should be able to click "Continue" on the welcome screen, and then see the error? And somehow the existing IDV outage code didn't pick this up? Would love a second set of eyes on this